package/post-install.sh

```diff
@@ -11,17 +11,25 @@ CUSTOM_INI_FILE_NAME='ddtrace-custom.ini'
 
 PATH="${PATH}:/usr/local/bin"
 
-if [[ -z "$DD_TRACE_PHP_BIN" ]]; then
-    DD_TRACE_PHP_BIN=$(command -v php)
+# We attempt in this order the following binary names:
+#    1. php
+#    2. php7 (some alpine versions install php 7.x from main repo to this binary)
+#    3. php5 (some alpine versions install php 5.x from main repo to this binary)
+if [ -z "$DD_TRACE_PHP_BIN" ]; then
+    DD_TRACE_PHP_BIN=$(command -v php || true)
+fi
+if [ -z "$DD_TRACE_PHP_BIN" ]; then
+    DD_TRACE_PHP_BIN=$(command -v php7 || true)
+fi
+if [ -z "$DD_TRACE_PHP_BIN" ]; then
+    DD_TRACE_PHP_BIN=$(command -v php5 || true)
 fi
 
-function invoke_php(){
-    # In case of .apk post-install hooks the script has no access the set of exported ENVS.
-    # In case when 1) users import large ini files (e.g. browsercap.ini) and 2) they set memory_limit using an env
-    # variable we would fail because of the memory limit reached. In order to avoid this we set a sane memory limit
-    # that should cause no problem in any machine when our tracer is installed.
-    $DD_TRACE_PHP_BIN -d memory_limit=128M $*
-}
+# In case of .apk post-install hooks the script has no access the set of exported ENVS.
+# When 1) users import large ini files (e.g. browsercap.ini) and 2) they set memory_limit using an env
+# variable we would fail because of the memory limit reached. In order to avoid this we set a sane memory limit
+# that should cause no problem in any machine when our tracer is installed.
+RUN_PHP="$DD_TRACE_PHP_BIN -d memory_limit=128M"
```

Comment:
This is an addition I was planning to do from a long time as it was reported by a user.

Reply:
What addition is this? Are you referring to quoting it?

Reply:
No this is a left-over/merge from a branch I am about to close.  Let me merge the two changes.